projects/igniteui-angular/src/lib/accordion/accordion.component.ts

```diff
@@ -231,14 +234,68 @@ export class IgxAccordionComponent implements AfterContentInit, AfterViewInit, O
         }
 
         this.panels.forEach(panel => {
-            if (!panel.collapsed) {
+            if (!panel.collapsed && !panel.header.disabled) {
                 panel.collapse();
             }
         });
     }
 
-    public handleKeydown(ev) {
+    public handleKeydown(event: KeyboardEvent, panel: IgxExpansionPanelComponent) {
+        const key = event.key.toLowerCase();
+        if (!(ACCORDION_NAVIGATION_KEYS.has(key))) {
+            return;
+        }
+        this.enabledPanels = this._panels.filter(p => !p.header.disabled);
+        event.preventDefault();
+        if (event.repeat) {
+            setTimeout(() => this.handleNavigation(event, panel), 1);
+        } else {
+            this.handleNavigation(event, panel);
+        }
+    }
+
+    private handleNavigation(event: KeyboardEvent, panel: IgxExpansionPanelComponent) {
+        switch (event.key.toLowerCase()) {
+            case 'home':
+                this.enabledPanels[0].header.elementRef.nativeElement.children[0].focus();
```

Comment:
Expose `panelheader.elementRef.nativeElement.children[0]` in a hidden internal getter on IgxExpansionPanelHeaderComponent